data-report-commons/src/main/java/commons/handlers/BulkTransformerHandler.java

```diff
@@ -92,22 +75,24 @@ protected Void processInput(KeyBatchRequestEvent input,
             .map(this::extractContent)
             .filter(keys -> !keys.isEmpty())
             .map(this::mapToIndexDocuments)
-            .map(this::aggregateNquads)
-            .map(this::removeNullCharacters)
-            .map(nquads -> attempt(() -> compress(nquads)).orElseThrow())
-            .map(this::persistNquads);
+            .map(this::processBatch)
+            .map(transformedData -> attempt(() -> compress(transformedData)).orElseThrow())
```

Comment:
why compress here?

Reply:
not a part of the pr, just curious

Reply:
Why compress at all or here in the code? I kept the original code from BulkTransformer, I can consider changing in in the next PR